BO4E/COM/Konfigurationsprodukt.cs

```diff
@@ -47,10 +47,20 @@ public class Konfigurationsprodukt : COM
     /// <summary>
     /// Auftraggebender Marktpartner
     /// </summary>
-    [JsonProperty(Required = Required.Default, Order = 4, PropertyName = "marktpartner")]
+    [JsonProperty(Required = Required.Default, Order = 4, PropertyName = "auftraggebenderMarktpartner")]
     [JsonPropertyOrder(4)]
-    [JsonPropertyName("marktpartner")]
+    [JsonPropertyName("auftraggebenderMarktpartner")]
     [ProtoMember(4)]
     [NonOfficial(NonOfficialCategory.CUSTOMER_REQUIREMENTS)]
-    public Marktteilnehmer? Marktpartner { get; set; }
+    public Marktteilnehmer? AuftraggebenderMarktpartner { get; set; }
+    
+    /// <summary>
+    /// Marktpartner für die die Produkt-Daten relevant sind.
```

Comment:
Kannst du hier noch den Satz aus dem MIG verkürzt hinzufügen, damit klar ist, warum wir es die empfangendeMarktrolle nennen?
```suggestion
    /// Marktrolle des Marktpartners für den die Produkt-Daten relevant sind.
```